.clang-format

```diff
@@ -0,0 +1,92 @@
+---
```

Comment:
this file shouldn't be here? (i.e. it will be copied from hpc-coding-convention)

Reply:
it's for ci, this is the file for clang-format 9.0, it is done like that in libsonata, I only copied it from this repository.